### DISCUSSION TITLE
Tablature?

### DISCUSSION THREAD

**Discussion by tresero** [opensheetmusicdisplay/opensheetmusicdisplay#1255] [Ideas]
Are there any plans for adding tablature and percussion staffs?

Much of my work is with guitar/other string instruments and percussion. I may be able to help, but I'm a database programmer, not much experience with typescript from a practical point of view. It is important to me and even though I'm retired from programming, I'll help if possible.

Here is a sample with tab and another with tab and percussion:
[compositor-se-equivico-tres.mxl.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/9834750/compositor-se-equivico-tres.mxl.zip)

[compositor-se-equivico.mxl.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/9834752/compositor-se-equivico.mxl.zip)

Here is what they should look like.
![compositor-se-equivico](https://user-images.githubusercontent.com/1476286/197085005-75bcd4d5-df58-4d78-b558-ccd09e5b9fb5.jpeg)

**Comment by sschmidTU** [comment]
This is already supported. Your files just lead to an error saying that 3 lines are an invalid number of lines. Might be that 3-line instruments are not supported in Vexflow, will take a look.

**Comment by tresero** [comment]
That was quick! Thanks. I am exporting from Finale 27 musicxml 3.1 (mainly because I still post stuff on Musescore and they don't support 4)

I'll be happy to send any other files you need.

**Reply by tresero** [reply]
It looks like there is some parameter for building other than standard guitar in tabstave. It doesn't look like it was ever answered though. This is just a very fast search and I can see in the code a way to init to other systems etc.

https://github.com/0xfe/vexflow/issues/356
